modules/vaos/spec/request/systems_request_spec.rb

```diff
@@ -25,63 +25,80 @@
   context 'with a loa3 user' do
     let(:user) { FactoryBot.create(:user, :loa3, ssn: '111223333') }
 
-    context 'with a valid GET systems response' do
-      it 'returns a 200 with the correct schema' do
-        VCR.use_cassette('vaos/systems/get_systems', match_requests_on: %i[method uri]) do
-          get '/v0/vaos/systems'
-
-          expect(response).to have_http_status(:ok)
-          expect(response.body).to be_a(String)
-          expect(response).to match_response_schema('vaos/systems')
+    describe 'GET /v0/vaos/systems' do
+      context 'with a valid GET systems response' do
+        it 'returns a 200 with the correct schema' do
+          VCR.use_cassette('vaos/systems/get_systems', match_requests_on: %i[method uri]) do
+            get '/v0/vaos/systems'
+
+            expect(response).to have_http_status(:ok)
+            expect(response.body).to be_a(String)
+            expect(response).to match_response_schema('vaos/systems')
+          end
         end
       end
-    end
 
-    context 'with a 403 response' do
-      it 'returns a VAOS 403 error response' do
-        VCR.use_cassette('vaos/systems/get_systems_403', match_requests_on: %i[method uri]) do
-          get '/v0/vaos/systems'
-          expect(response).to have_http_status(:forbidden)
-          expect(JSON.parse(response.body)['errors'].first['code']).to eq('VAOS_403')
-          expect(response).to match_response_schema('errors')
+      context 'with a 403 response' do
+        it 'returns a VAOS 403 error response' do
+          VCR.use_cassette('vaos/systems/get_systems_403', match_requests_on: %i[method uri]) do
+            get '/v0/vaos/systems'
+
+            expect(response).to have_http_status(:forbidden)
+            expect(JSON.parse(response.body)['errors'].first['code']).to eq('VAOS_403')
```

Comment:
Would a helper method here DRY things up? Something like

```
def response_error(response)
	JSON.parse(response.body)['errors'].first['code'])
end
```

Would make it a lot simpler to read

Reply:
Looks like someone already did something similar with the response schema matcher below.